File: 58-LengthOfLastWord/58-LengthOfLastWord.js
// Last updated: 3/6/2026, 2:09:40 PM
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let res = "";
    let length = false;
    for(let i=s.length-1;i>=0;i--){
        if(s[i]!==" "){
            length = true;
            res = res + s[i];
        }
        else if(length){
            break;
        }
    }
    return res.length;
};

File: 58-LengthOfLastWord/58-LengthOfLastWord.cpp
// Last updated: 7/17/2026, 4:28:12 PM
class Solution {
public:
    int lengthOfLastWord(string s) {
       int len = 0;

       for(int i=s.length()-1;i>=0;i--){
        if(s[i]!=' '){
            len++;
        }
        else if(len>0){
            break;
        }
       }
 return len;

    }
};